pipelines/legionPipeline.groovy

```diff
@@ -123,13 +127,15 @@ def destroyGcpCluster() {
                         docker.image("${env.param_docker_repo}/k8s-terraform:${env.param_legion_infra_version}").inside("-e GOOGLE_CREDENTIALS=${gcpCredential} -e CLUSTER_NAME=${env.param_cluster_name} -u root") {
                             stage('Extract Hiera data') {
                                 extractHiera("json")
+                                gcp_zone = sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)
```

Comment:
We can extract `sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)` to a separated function

Reply:
can't see a lot of use in function because getting the value will be almost the same as it is now - we need profile path, value and function just another syntax, but the same amount of code +function

Reply:
`gcp_zone = sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)`
->
`gcp_zone = profile('location')`
?

Reply:
`gcp_zone = sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)`
->
`gcp_zone = profile('location')`
?

Reply:
`gcp_zone = sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)`
->
`gcp_zone = profile('location')`
?

Reply:
that assumes hardcoded cluster profile location.

Reply:
It's better because now you hardcode the cluster profile location at 8 places. And we will continue to make hardcoded location later.